conf/app.routes

```diff
@@ -1,3 +1 @@
 # microservice specific routes
```

Comment:
it has no routes?? How does the service interact with the stubs?

Reply:
This needs removing as routes are in prod.routes